variables.tf

```diff
@@ -1334,3 +1334,14 @@ variable "skip_all_s2s_auth_policies" {
 }
 
 ##############################################################################
+
+##############################################################################
+# CBR variables
+##############################################################################
+variable "existing_vpc_cbr_zone_id" {
+  type        = string
+  description = "Existing VPC CBR zone id"
```

Comment:
Perhaps explain a little more about what happens when user passes a value for this